xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -67,9 +67,11 @@ CVideoPlayerVideo::CVideoPlayerVideo(CDVDClock* pClock
   m_iDroppedRequest = 0;
   m_fForcedAspectRatio = 0;
 
-  // 128 MB allows max bitrate of 128 Mbit/s (e.g. UHD Blu-Ray) during 8 seconds
-  m_messageQueue.SetMaxDataSize(128 * 1024 * 1024);
-  m_messageQueue.SetMaxTimeSize(8.0);
+  m_messageQueueTimeSize = messageQueueTimeSize;
```

Comment:
you can probably use the initializer list

Reply:
done